### ISSUE TITLE
RuntimeError: Failed to calculate an energy for O

### ISSUE THREAD

**Issue by AiPacino9** [duartegroup/mlp-train#124]
Hi, i met some problems when running examples/DA_paper/training/explicit/cis_endo_TS_wB97M.xyz
The following are not valid: tmp_orca.inp
Cannot start the screen reader because it cannot connect to the Desktop.

Traceback (most recent call last):
  File "/home/aipacino/code/mlp-train/examples/DA_paper/training/explicit/endo_ace_ex.py", line 280, in <module>
    Water_mlp.al_train(
  File "/home/aipacino/code/mlp-train/mlptrain/potentials/_base.py", line 196, in al_train
    al_train(self, method_name=method_name, **kwargs)
  File "/home/aipacino/code/mlp-train/mlptrain/training/active.py", line 187, in train
    mlp.set_atomic_energies(method_name=method_name)
  File "/home/aipacino/code/mlp-train/mlptrain/potentials/_base.py", line 318, in set_atomic_energies
    raise RuntimeError(
RuntimeError: Failed to calculate an energy for O

How can i solve it. Thanks!
